sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Azure.Monitor.OpenTelemetry.AspNetCore.csproj

```diff
@@ -23,10 +23,10 @@
     <!-- Depending on monthly deliverables, we may switch between PackageReference or ProjectReference. Keeping both here to make the switch easier. -->
 
     <!-- FOR PUBLIC RELEASES, MUST USE PackageReference. THIS REQUIRES A STAGGERED RELEASE IF SHIPPING A NEW EXPORTER. -->
-    <PackageReference Include="Azure.Monitor.OpenTelemetry.Exporter" />
+    <!--<PackageReference Include="Azure.Monitor.OpenTelemetry.Exporter" />-->
```

Comment:
Could you please remind me why this is getting changed?

Reply:
The `OpenTelemetry` base SDK dependency comes transitively from the Exporter.